src/com/pentalog/pentastagiu/homework2/Client.java

```diff
@@ -0,0 +1,30 @@
+package com.pentalog.pentastagiu.homework2;
+
+public class Client {
+    String FirstName;
+    String LastName;
+    double debit = 0;
+
+    public void deposit_money( int sum)
+    {
+        debit=debit+sum;
```

Comment:
dar daca suma pe care vrea sa o depuna e negativa? ar trebui sa faci niste validari pentru a te asigura ca e corect ce se intampla in metoda; poti ajunge cu o suma negativa in cont daca nu faci aceste validari